app/forms/openid_connect_authorize_form.rb

```diff
@@ -114,6 +115,10 @@ def aal
       Saml::Idp::Constants::DEFAULT_AAL_AUTHN_CONTEXT_CLASSREF
   end
 
+  def aal_value
+    aal_values.sort.max
+  end
```

Comment:
yes I think with all the different versions of the aal value floating around its going to be really confusing without clearer variable naming

Reply:
My only reason to not change the name is because line `def aal` on 113 and `def ial` on 105 also use the .max without specifically calling out "highest level" or similar.